core/catkin_ws/src/custom_msgs/msg/Object.msg

```diff
@@ -0,0 +1,10 @@
+std_msgs/Header header
```

Comment:
Lets call this something like CVObject, Object by itself is common enough to make this confusing

Reply:
Sounds good -- we'll push that change soon.